contracts/BatchExchange.sol

```diff
@@ -398,7 +398,7 @@ contract BatchExchange is EpochTokenLocker {
             address user = allUsers.first();
```

Comment:
I think `getEncodedOrders` also needs to support pagination, or else the driver will run into the same problems when it tries to get the order book to pass to the solver.

Reply:
I think the requirement is only needed for the front-end as the solver can just point to another fullnode/our fullnode, which does not have the same restrictions as infura

Reply:
Oh good point!